blocks/breadcrumbs/breadcrumbs.css

```diff
@@ -0,0 +1,64 @@
+.section.breadcrumbs-container {
```

Comment:
This is style leaking out from the block to its parent container. Can we avoid this? Else it would be mandatory to place `breadcrumb` block always in its exclusive section.

Reply:
+1
its should be 

`.block.<classname>`

Reply:
I had to add this specific selector to override the style coming from the style.css file where we are adding margin of 2rem default on all sections 
/* sections */
main .section {
  margin: 2rem 0;
}

I wanted to avoid adding negative margin by selecting nested .block.breadcrumbs node.

Reply:
How about we change 
```
main .section {
margin: 2rem 0;
}
```
this only if `padded` class is present in the section instead of all section ?
Do we any reason to belive that main .section  needs this margin ? for all section ?

Reply:
@vivesing then let's remove it from global css. We can add margin property such as `main .section.spaced` for the individual section that needs the margin.

Reply:
heade and footer are not part of main anyway and rest all are using `padded` style anyway.

Reply:
Since I just had to remove the top margin but needed the rest, moving ahead with negative margin for breadcrumbs. This will keep rest of the blocks intact.